src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(String operation, String fruit, int quantity) {
+        this.operation = Arrays.stream(Operation.values())
+                .filter(o -> o.getOperation().equals(operation))
+                .findFirst()
+                .get();
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
it looks like a hardcoded constructor
maybe you can create a separate method with this logic? in Operation inner class for example
it's up to you of course)